controller-server/src/main/java/com/yahoo/vespa/hosted/controller/ApplicationController.java

```diff
@@ -517,7 +517,8 @@ private ActivateResult deploy(ApplicationId application, ApplicationPackage appl
         } finally {
             // Even if prepare fails, a load balancer may have been provisioned. Always refresh routing policies so that
             // any DNS updates can be propagated as early as possible.
-            controller.routing().policies().refresh(application, applicationPackage.deploymentSpec(), zone);
+            if ( ! application.instance().isTester())
```

Comment:
This is not necessary as LoadBalancerProvisioner never provision load balancers testers.

Reply:
Great.